faq.md

```diff
@@ -372,6 +372,12 @@ The term _Manufacturer_ is defined in [Article 3(13)][] of the CRA:
 
 Yes, a _manufacturer_ can also be an _open-source software steward_, but it cannot be both the _manufacturer_ and _open-source software steward_ of the same project.
 
+In 2024 Benjamin Bögel of the European Commission gave a FOSDEM presentation and specifically gave, as examples of stewards, “companies that build [OSS] for their use [for integration into their own products] but make [the OSS] public.” [^EC@FOSDEM24][]. This only makes sense if an organization can be a manufacturer for one program and a steward for another.
+
+Mike Bursell, Co-chair, OpenSSF Cyber Policy Working Group (WG), believes that “if the organisation both sells a PDE that uses [some] open source project, and [separately] hosts [that open source project], supports it, and provides updates and patches for the community [that’s a real community]… then that would put [the organization] in both categories [depending on which software is being discussed].” [Bursell2025][]
```

Comment:
I agree with the substance. But does it make sense to provide non-authoritative sources (Mike) as a citation?

Reply:
Yes, I think it's important to include the quote from Mike here. The statement from Mike, while not authoritative like a statement from the EC, shows that someone with expertise (and thus is likely to know the correct answer) agrees with this interpretation.  Including this quote undercuts the argument that "an EC member's statement was taken out of context, no one believes that".

I think the overall goal for FAQ responses should be to provide strong evidence that the answers given here are correct (not just unjustified answers). This question is one of the most common questions we (OpenSSF) have been asked. I think providing an array of evidence is always best, as that provides stronger support. Providing strong multi-source evidence in this case is especially important.